HotKey/HotKey_MainFolder/ModKeys.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HotKey_MainFolder
+{
+    [Flags]
```

Comment:
Do we want to use this for the code I uploaded? Repetitive for us to create it twice. See my comment on storing the keys in one main class.